cli/Cargo.toml

```diff
@@ -111,7 +111,7 @@ jj-cli = { path = ".", features = ["test-fakes"], default-features = false }
 [features]
 default = ["watchman"]
 bench = ["dep:criterion"]
-packaging = []
+packaging = ["jj-lib/gix-max-performance"]
```

Comment:
nit: might be better to alias `gix-max-performance` through cli/Cargo.toml (like `watchman`)?

Reply:
Makes sense, I added the top-level `gix-max-performance` feature to `cli/Cargo.toml` and had `packaging` reference it.